linter.ini

```diff
@@ -1,6 +1,6 @@
 [flake8]
 ignore = E252,W504,W503
-max-line-length = 120
+max-line-length = 500
```

Comment:
Let's not mod this. If you need help with figuring out how to best get a line to fit into length requirements, we can help!